v3/src/models/history/tree-types.ts

```diff
@@ -50,6 +50,10 @@ export function isChildOfUndoRedo(actionCall?: IActionContext) {
   return false
 }
 
+export function isUndoingOrRedoing() {
+  return isChildOfUndoRedo(getRunningActionContext())
+}
```

Comment:
This is currently unused. I added it thinking it would be necessary but ended up not needing it, but it seems like it could be useful so I'm leaving it in for now.